.infrastructure/ingress/ingress.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
There is an extra space in the `apiVersion` declaration. It should be `apiVersion: networking.k8s.io/v1` without any spaces before or after the colon.